lib/lrug_helpers.rb

```diff
@@ -131,7 +131,7 @@ def sponsor_logo(sponsor_name, size: 'sidebar')
     if sponsor
       link_text =
         if sponsor.logo? && sponsor.logo[size]
-          %{<img src="#{sponsor.logo[size].url}" width="#{sponsor.logo[size].width}" height="#{sponsor.logo[size].height}" alt="#{sponsor.name}" title="#{sponsor.name} Logo"/>}
+          %{<img src="#{sponsor.logo[size].url}" width="#{sponsor.logo[size].width}" height="#{sponsor.logo[size].height}" alt="#{sponsor.name}" title="#{sponsor.name} Logo" loading="lazy"/>}
```

Comment:
I added native lazy loading, it appears to drop our homepage loading time quite nicely so I've kept it in.